src/ui/components/shared/UserSettingsModal/PreferencesSettings.tsx

```diff
@@ -68,15 +68,11 @@ function NotificationPreferences({
 function PrivacyPreferences() {
   const [disableLogRocket, setDisableLogRocket] = useState<boolean>();
   const { userSettings, loading } = hooks.useGetUserSettings();
-  const updateLogRocket = hooks.useUpdateUserSetting("disableLogRocket");
+  const { updateUserPreferences } = hooks.useUpdateUserPreferences();
 
   const toggle = (newValue: boolean) => {
     setDisableLogRocket(newValue);
-    updateLogRocket({
-      variables: {
-        newValue,
-      },
-    });
+    updateUserPreferences({ variables: { preferences: { disableLogRocket: newValue } } });
```

Comment:
What are the implications here?

That we should only send preferences that have been _changed_ by the user within a session (rather than sending all of the current prefs object)?

Reply:
I just wanted to point out that it's not necessary to send all current prefs (in case that wasn't obvious), we can keep sending only the changed ones (which is what I would recommend).